packages/actions/test/unit/security.test.ts

```
@@ -0,0 +1,112 @@
+        user1.uid = currentAuthenticatedUser.uid
+
+        // create 2nd user
+        await createNewFirebaseUserWithEmailAndPw(userApp, user2.data.email, user2Pwd)
+        await sleep(5000)
+
+        // Retrieve the current auth user in Firebase.
+        currentAuthenticatedUser = getCurrentFirebaseAuthUser(userApp)
+        user2.uid = currentAuthenticatedUser.uid
+
+        // create the coordinator
+        await createNewFirebaseUserWithEmailAndPw(userApp, user3.data.email, user3Pwd)
+        await sleep(5000)
+
+        currentAuthenticatedUser = getCurrentFirebaseAuthUser(userApp)
+        user3.uid = currentAuthenticatedUser.uid
+
+        // add coordinator privileges
+        await addCoordinatorPrivileges(adminAuth, user3.uid)
+    })
+
+    it("should work as expected and return the data for the same user", async () => {
+        // login as user1
+        await signInWithEmailAndPassword(userAuth, user1.data.email, user1Pwd)
+        const userDoc = await getDocumentById(userFirestore, "users", user1.uid)
+        expect(userDoc.data()).to.not.be.null
+    })
+
+    it("should allow anyone to query the ceremony collection", async () => {
+        // login as user2
+        await signInWithEmailAndPassword(userAuth, user2.data.email, user2Pwd)
+        // query the ceremonies collection
+        expect(await queryCollection(userFirestore, "ceremonies", [where("description", "!=", "")])).to.not.throw
+    })
+
+    it("should allow the coordinator to read another user's document", async () => {
+        // login as coordinator
+        await signInWithEmailAndPassword(userAuth, user3.data.email, user3Pwd)
+        // retrieve the document of another user
+        const userDoc = await getDocumentById(userFirestore, "users", user1.uid)
+        const data = userDoc.data()
+        expect(data).to.not.be.null
+    })
+
+    it("should not return another user's document if not authenticated as a coordinator", async () => {
```

nit: "should throw an error if a non-coordinator user ..."